core/src/main/java/com/linecorp/armeria/common/util/TimeoutController.java

```diff
@@ -69,11 +69,32 @@
     /**
      * Cancels the current timeout scheduled. You can schedule a new timeout with
      * {@link #scheduleTimeout(long)} if the current timeout is canceled successfully.
-     * @return {@code true} if the current timeout is canceled or no timeout was scheduled previously.
-     *         {@code false} if the timeout has been triggered already.
+     * @return {@code true} if the current timeout is canceled.
+     *         {@code false} if the timeout has been triggered already or no timeout was scheduled previously.
      */
     boolean cancelTimeout();
 
+    /**
+     * Returns whether the timeout is scheduled or not.
+     *
+     * @return {@code true} the timeout is scheduled now.
+     *         {@code false} if the timeout has been triggered already or no timeout was scheduled previously.
+     */
+    boolean isScheduled();
```

Comment:
It seems like this method is not used anywhere? Maybe OK to remove?

Reply:
Yes, it don't use anywhere. I was considering to add this or not. I've already added other boolean getters for `TIMED_OUT` and `DISABLED`.
So I thought it would be more natural for adding boolean getters for `SCHEDULED`.
But it seems overkill. Let me remove the unused code. 😀